app/helpers/rademade_admin/field_helper.rb

```diff
@@ -9,21 +9,25 @@ module RademadeAdmin::FieldHelper
   # @return [String]
   #
   def display_item_value(item, data_item)
+    return data_item.list_preview_handler.call(item) if data_item.list_preview_handler
     value = item.send(data_item.list_preview_accessor)
-    if data_item.has_relation?
-      # rm_todo extract method
-      if data_item.relation.has_many?
-        link_to data_item.label, admin_related_item(item, data_item.getter)
-      else
-        link_to value.to_s, admin_edit_uri(value) unless value.nil?
-      end
-    elsif data_item.has_uploader?
-      RademadeAdmin::Upload::PreviewService.new(value).uploaded_file_html
+    return display_upload_item(data_item, value) if data_item.has_uploader?
+    return display_related_item(data_item, value) if data_item.has_relation?
+    value.to_s.html_safe
+  end
+
+  def display_related_item(data_item, value)
+    if data_item.relation.has_many?
+      link_to(data_item.label, admin_related_item(item, data_item.getter))
     else
-      value.to_s.html_safe
+      link_to(value.to_s, admin_edit_uri(value)) unless value.nil?
     end
   end
 
+  def display_upload_item(data_item, value)
```

Comment:
Unused method argument - `data_item`. If it's necessary, use `_` or `_data_item` as an argument name to indicate that it won't be used.